app/views/components/_server_summary.html.erb

```diff
@@ -7,9 +7,9 @@ synchronized ||= false
     <%= render partial: "components/starburst" %>
   </div>
   <% unless noHeader %>
-    <a id="carouselPanel" href="/servers/<%= server.id %>">
+    <a id="carouselPanel" href="/servers/<%= server.id %>" title="<%=server.name%>">
       <div class="server-summary-details">
-        <span class="server-name"><%= server.name %> - </span>
+        <span class="server-name"><%= truncate(server.name, length: 25) %>:</span>
```

Comment:
Do we want to add ellipses (...) when we truncate the server name? If so, there might be a CSS-y way to do both truncation of the name and addition of the ellipses.